Otsu_Thresholding.py

```diff
@@ -0,0 +1,65 @@
+import matplotlib.pyplot as plt
+from skimage.io import imread
+from skimage.filters import threshold_otsu
+
+# -*- coding: utf-8 -*-
+"""
+Created on Wed May  1 11:08:50 2019
+Global Otsu thresholding
+@author: mostly Luis
+"""
+
+def import_image(path, name):
```

Comment:
"path" commonly means the actual path to the file, something like 
`C:\users\bradypus\pictures\holiday.jpg`
What you mean in this context is the directory.

In line 14: What you mean by raw_image_directory is the actual path to the file.